doc/reposync.rst

```diff
@@ -67,6 +67,9 @@ All general DNF options are accepted. Namely, the ``--repoid`` option can be use
 ``-p <download-path>, --download-path=<download-path>``
     Root path under which the downloaded repositories are stored, relative to the current working directory. Defaults to the current working directory. Every downloaded repository has a subdirectory named after its ID under this path.
 
+``--safe-write-path``
+    Root path that is considered safe for writing. If not specified it defaults to download path of the repository. Useful for repositories that use relative locations of packages out of repository directory (e.g. "../packages_store/foo.rpm"). Use with care, any file under the ``safe-write-path`` can be overwritten. Can be only used when syncing a single repository.
```

Comment:
Hmm, I am just thinking if not to explicitly mention that also any subdirectory within this path is considered safe. It was not clear to me when reading this, but I realized that when reading the implementation. Maybe it's just lack of the morning coffee though...